src/errors.js

```diff
@@ -63,10 +63,14 @@ exports.Forbidden = class Forbidden extends HttpError {
 
 exports.DependencyError = class DependencyError extends HttpError {
     constructor (e, connector) {
-        super(503, 'DEPENDENCY_UNAVAILABLE', 'Service dependency unavailable', {
-            name: connector.getName(),
-            impl: connector.getImpl()
-        });
+        super(
+            503,
+            'DEPENDENCY_UNAVAILABLE',
+            'Internal service dependency unavailable, please contact Red Hat support: https://access.redhat.com/support/cases/', {
```

Comment:
@aleccohan since this type of issue is usually transient, maybe change the text to something like
'Internal service dependency is temporarily unavailable. If the issue persists please contact Red Hat support: https://access.redhat.com/support/cases/'

Reply:
Will do! Changing now!